corehq/apps/linked_domain/dbaccessors.py

```diff
@@ -32,6 +32,17 @@ def is_active_downstream_domain(domain):
     return DomainLink.objects.filter(linked_domain=domain).exists()
 
 
+def is_active_link(upstream_domain, downstream_domain):
+    return bool(get_active_domain_link(upstream_domain, downstream_domain))
+
+
+def get_active_domain_link(upstream_domain, downstream_domain):
+    try:
+        return DomainLink.objects.get(master_domain=upstream_domain, linked_domain=downstream_domain)
```

Comment:
How about keeping this as `filter` and returning `first`? Then you'd have a one-liner.